ospo-mindmap/Content/ospomindmap.md

```diff
@@ -30,6 +30,11 @@ efficient through sharing costs and innovation for industry-specific needs
 - `Definition` Work on foundational technology problems that cross industries
 - `Example:` Bloomberg worked with Microsoft to make contributions to TypeScript
 
+### Standards Collaborative
+
+- `Definition` Support relevant standards in open source projects and provide feedback to corresponding standards organizations
+- `Example:` Let's Encrypt community collaborates with IETF standards community on ACME, TLS, and other work related to X.509 certificates
```

Comment:
I agree that OSPOs may have reasons to collaborate with regulatory bodies, non profits, and private entities; however, when that collaboration pertains to standards, it is better for the collaboration to occur within the corresponding standards organization than for the OSPOs to work directly with regulatory bodies outside of the standards organizations.

Reply:
I see your point and in most cases, it happens as you describe. However, AFAIK, the way that Governmental OSPOs operate is slightly different: They provide feedback to regulatory bodies instead of standards organizations. But I'm not an expert in the sector so I'm not 100% sure 🤔

Reply:
I am Seth Newberry with the JDF, the standards arm of the LF.  We would be happy to share our thoughts on how Standards might be incorporated with an OSPO.  In the first place, standards are quite different from Open Source Code, especially in the use of license commitments for IPR.  When you want to interact with other Standards Development Organizations (SDO), you need to make sure the license terms you are using are compatible with the Standards Body you want to interact with, especially if you are incorporating any content from the outside SDO.  If their IPR terms are different than yours, it has the risk of contaminating your specification with incompatible IPR.  Also ideas can contain IPR.  So while an OSPO tends to thing in terms of Github pull requests in the host repo, a conversation with an SDO can impart a concept that may be patent encumbered.  So you need to think carefully about how you take (or give) contributions from another SDO.  This is a pretty detailed topic, too large for a single comment, but JDF is happy to help you develop policy on this topic.

Reply:
I agree it is a big and important topic. Is there an opportunity for a BoF on this at the upcoming OSSEU conference?